numpy/_core/tests/test_datetime.py

```diff
@@ -581,6 +581,15 @@ def test_timedelta_scalar_construction_units(self):
         assert_equal(np.datetime64('now').dtype,
                      np.dtype('M8[s]'))
 
+    @pytest.mark.parametrize('args', [(), ('s',)])
```

Comment:
In case of non-NaT values, with `'ns'` and `'M'` you get  `_.item() -> int`, but with `'s'` you het `_.item() -> datetime.timedelta`. If that's irrelevant for the C changes here, then feel free to ignore me, but otherwise something like this could help:

```suggestion
    @pytest.mark.parametrize('args', [(), ('ns', 's', 'M')])
```